### PR TITLE
Added new option to version compare manager

### PR DESCRIPTION
**Pull request by CalebGerman**

### CODE REVIEW

**Review by sarah-pontier — APPROVED** [review]

File: packages/changed-elements-react/src/api/VersionCompare.ts [hunk 1]
```diff
@@ -33,6 +34,12 @@ export interface VersionCompareOptions {
    */
   changedElementsApiBaseUrl?: string | undefined;
 
+  /** Enable or disable display of side by side toggle in property comparison table */
```
Comment (mattbjordan) [inline packages/changed-elements-react/src/api/VersionCompare.ts:37]:
```suggestion
  /** Enable or disable display of side by side toggle in property comparison table. */
```

File: packages/changed-elements-react/src/api/VersionCompare.ts [hunk 2]
```diff
@@ -33,6 +34,12 @@ export interface VersionCompareOptions {
...
+  displaySideBySideToggle?: boolean | undefined;
+
+  /** Optional prop for a user supplied component to handle managing named versions.*/
```
Comment (mattbjordan) [inline packages/changed-elements-react/src/api/VersionCompare.ts:40]:
```suggestion
  /** Optional prop for a user supplied component to handle managing named versions. */
```

File: packages/changed-elements-react/src/api/VersionCompare.ts [hunk 3]
```diff
@@ -33,6 +34,12 @@ export interface VersionCompareOptions {
    */
   changedElementsApiBaseUrl?: string | undefined;
 
+  /** Enable or disable display of side by side toggle in property comparison table */
+  displaySideBySideToggle?: boolean | undefined;
```
Comment (mattbjordan) [inline packages/changed-elements-react/src/api/VersionCompare.ts:38]: Does this prop ever get passed down to the component that controls the side by side view?

**Review by mattbjordan — CHANGES_REQUESTED** [review]

File: packages/changed-elements-react/src/api/VersionCompare.ts (hunk 3 above)
Comment (CalebGerman) [reply packages/changed-elements-react/src/api/VersionCompare.ts:38]: Good point i was thinking about it from the pineapple consumption point of view

**Pull request merged**